packages/plugin-ext/src/main/browser/workspace-main.ts

```diff
@@ -63,13 +63,13 @@ export class WorkspaceMainImpl implements WorkspaceMain {
 
     notifyWorkspaceFoldersChanged(): void {
         if (this.roots && this.roots.length) {
```

Comment:
not part of your change - `this.roots` won't be null or undefined, and therefore checking it is not necessary :)

Reply:
@elaihau Thanks for the review,
I'm going to open another PR where this checking is removed.